kernelci/build.py

```diff
@@ -1228,6 +1290,9 @@ def run(self, jopt=None, verbose=False, opts=None):
         *verbose* is whether the build output should be shown
         """
         bmeta = self._meta.get('bmeta')
+        # Fetch linux firmware if this option enabled
+        # it means we need to have it ready to be embedded during build
+
```

Comment:
```suggestion
```
Guess this was left from the previous version before splitting the step into a separate one?

Reply:
Yes, sorry forgot to remove, at moment i am trying to get concept working on staging, then i will work on improving code and wrapping config so it will have better readability.